test/models/application_test.rb

```diff
@@ -7,14 +7,18 @@ def test_it_has_a_selection_of_statuses
   end
 
   def test_it_returns_applications_with_different_statuses
-    application1 = create(:application)
-    application2 = create(:application, status: "in-progress")
-    application3 = create(:application, status: "applied")
-    application4 = create(:application, status: "closed")
+    to_apply_to_application_a = create(:application)
+    to_apply_to_application_b = create(:application)
```

Comment:
What's the deal with all these useless assignments? Is there a Hound-approved way of doing this?